src/@dvcorg/gatsby-theme-iterative/components/Documentation/Layout/SidebarMenu/icons/index.ts

```diff
@@ -3,11 +3,13 @@ import defaultIcons, {
 } from '@dvcorg/gatsby-theme-iterative/src/components/Documentation/Layout/SidebarMenu/icons'
 import { ReactComponent as CMLIcon } from './cml-icon.svg'
 import { ReactComponent as StudioIcon } from './studio-icon.svg'
+import { ReactComponent as VsCodeIcon } from '../../../../../../../../static/img/vscode-icon.svg'
```

Comment:
Would look better in a grayed-out version, but we can only use the officially logo legally :/